### PR TITLE
Update storm example to use v1

### PR DESCRIPTION
**Pull request by krousey**
#9286

@caesarxuchao @bgrant0607 @nikhiljindal

### CODE REVIEW

File: examples/storm/storm-nimbus-service.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
```
Comment (caesarxuchao) [inline examples/storm/storm-nimbus-service.json]: Could you remove the targetPort in this file?

File: examples/storm/zookeeper-service.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
```
Comment (caesarxuchao) [inline examples/storm/zookeeper-service.json]: Could you remove the targetPort in this file? Thanks.

**Comment by caesarxuchao** [comment]
Two nits. Otherwise LGTM. Thanks!

**Comment by krousey** [comment]
Done.

**Comment by caesarxuchao** [comment]
LGTM. Thanks!

**Pull request merged**